ibm/service/power/resource_ibm_pi_image.go

```diff
@@ -111,10 +111,11 @@ func ResourceIBMPIImage() *schema.Resource {
 				ForceNew:      true,
 			},
 			helpers.PIImageStorageType: {
```

Comment:
Are we not relabeling helpers.PIImageStorageType to helpers.PIImageStorageTier to match the new CLI wording?

Reply:
We discussed this — We will have to open a jira for all those changes.